### PR TITLE
[mind UI] Soulblades HUD. Goodbye gui_icons

### PR DESCRIPTION
**Pull request by DeityLink**
With this PR, gui_icons are finally gone.
RIP

imageimage

2014 - 2021

image
The only noticeable change gameplay wise is a more readable blood count number.

After that, it'll be time to design brand new UIs, and I have Cult in my sights. The Cult UI update will be split in two.

The first will be tied with a revamp of the new player experience, with new cultist sub-roles akin to chief cultist that will help promote teamwork and more importantly pair newbie cultists with more experienced ones.

The second one will be the long awaited Rituals Update that will completely replace the ACT system.

### CODE REVIEW

**Comment by drawsstuff** [comment]
Go king go.

File: code/_onclick/mindUI/soulblade.dm [hunk 1]
```diff
@@ -0,0 +1,73 @@
+
+////////////////////////////////////////////////////////////////////
+//																  //
+//					   SOULBLADE (BLOOD GAUGE)					  //
+//																  //
+////////////////////////////////////////////////////////////////////
+
+/datum/mind_ui/soulblade
+	uniqueID = "Soulblade"
+	x = "LEFT"
```
Comment (Kurfursten) [inline code/_onclick/mindUI/soulblade.dm:10]: I am aware we haven't required this before, but this should probably just be a define. (for all of them, like y's BOTTOM too)

Comment (DeityLink) [reply code/_onclick/mindUI/soulblade.dm:10]: screen_loc are strings though, ones that are composed of several parts either. I think it's best to keep the elements that compose it as explicit as possible. Or do you think we should replace every "LEFT:16, CENTER+3:8" with "[LEFT]:16, [CENTER]+3:8" ? I don't think it's sensible.

Comment (DeityLink) [reply code/_onclick/mindUI/soulblade.dm:10]: should I rename the var "screen_loc_x_base" so its purpose is more clear?

Comment (DeityLink) [reply code/_onclick/mindUI/soulblade.dm:10]: And lastly for the love of god lets not go back to shit like: 
image

defines used only once, defined in another file which makes you needlessly jump between files and makes designing/adjusting UIs more convoluted for little actual gain. (I'm having my morning coffee, I probably sound much more annoyed than I really am)

Comment (Kurfursten) [reply code/_onclick/mindUI/soulblade.dm:10]: No, it's pointless if they are cobbled like this, Defines are supposed to preempt typos but it won't help here.

**Pull request merged**